pyquil/quilatom.py

```diff
@@ -5,20 +5,21 @@ class QuilAtom(object):
     """
```

Comment:
Prime candidate for an ABC when we're upgraded

Reply:
:+1: